dev-python/moviepy/moviepy-0.2.2.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Video editing with Python"
+HOMEPAGE="http://zulko.github.io/moviepy/"
+SRC_URI="https://github.com/Zulko/moviepy/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE=""
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	"
```

Comment:
do I need to define `python_test()` explicitly? I am trying to build it with `FEATURES="test"` and I only get a one-liner telling me that testing was completed. Doesn't change anything if I define:

```
python_test() {
    py.test --verbose || die
}
```

(as I see many others do with pytest), or:

```
python_test() {
    tests/test.py --verbose || die
}
```

which is where I actually see the test.